packages/core/src/link.js

```diff
@@ -5,37 +5,37 @@ import * as API from '@ucanto/interface'
  * @template {number} Code
  * @template {number} Alg
  * @param {Code} code
- * @param {API.MultihashDigest<Alg>} digest
- * @return {API.Link<unknown, Code, Alg, 1> & CID}
+ * @param {API.UCAN.MultihashDigest<Alg>} digest
+ * @return {API.Link<any, Alg>}
```

Comment:
We should not introduce `any`, here it's really `unknown`.

```suggestion
 * @return {API.Link<unknown, Alg>}
```